.gitignore

```diff
@@ -53,3 +64,6 @@ dependency-reduced-pom.xml
 
 # Dart
 executors/dart/.dart_tool/
+executors/dart_native
+executors/dart_web
```

Comment:
If there are important source files in either `executors/dart_native` or `executors/dart_web`, then we shouldn't add the whole directory like this to the ignore file.